foundation-models/openai/src/test/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClientTest.java

```diff
@@ -373,6 +374,22 @@ void embedding() {
                       {"input":["Hello World"]}""")));
   }
 
+  @Test
+  void testThrowsOnContentFilter() {
```

Comment:
Was untested before. Also I needed to add a test, because after optimizing away some code the coverage dropped below the threshold 🫠